lib/child_process.js

```diff
@@ -440,9 +440,11 @@ function normalizeSpawnArguments(file, args, options) {
     const command = [file].concat(args).join(' ');
 
     if (process.platform === 'win32') {
-      file = typeof options.shell === 'string' ? options.shell :
-              process.env.comspec || 'cmd.exe';
-      args = ['/d', '/s', '/c', '"' + command + '"'];
+      if (typeof options.shell === 'string')
+        file = options.shell;
+      else
+        file = process.env.comspec || 'cmd.exe';
```

Comment:
The same way a user can remove `cmd.exe` from the `Path` or from the system completely.
IMHO we should deprecate the `|| 'cmd.exe'` part.

Ref: https://github.com/nodejs/node/issues/14156
Ref: https://github.com/nodejs/node/issues/14157